### PR TITLE
fix update options

### PR DESCRIPTION
**Pull request by stevemao**
fixes https://github.com/superRaytin/react-monaco-editor/issues/156

### CODE REVIEW

File: src/editor.js [hunk 1]
```diff
@@ -71,7 +74,7 @@ class MonacoEditor extends React.Component {
     const { language, theme, options } = this.props;
     if (this.containerElement) {
       // Before initializing monaco editor
-      Object.assign(options, this.editorWillMount());
+      Object.assign({}, options, this.editorWillMount());
```
Comment (domoritz) [inline src/editor.js]: Can you explain this change to me?

Comment (stevemao) [reply src/editor.js]: You can mutate `options` object. `options` object is coming from the `this.props` so you could mutate your props.

Comment (domoritz) [reply src/editor.js]: Yeah, I understand that. What I wonder is whether `this.editorWillMount()` should be merged into the options or not. It looks very intentional to me and so I just wondered what was happening here.

Comment (stevemao) [reply src/editor.js]: Let me revert that change since this is not relevant to what I'm trying to fix :)

Comment (domoritz) [reply src/editor.js]: I'd still love to understand it and fix it if necessary. Could you look into it and send a PR if we want to change this?

Comment (domoritz) [reply src/editor.js]: Unfortunately, I didn't write the code and would have to dig into the code myself.

Comment (stevemao) [reply src/editor.js]: Created an issue https://github.com/superRaytin/react-monaco-editor/issues/163

**Comment by stevemao** [comment]
@domoritz could you make a new release?

**Comment by domoritz** [comment]
I'll make one later today.

**Pull request merged**